code/index.html

```diff
@@ -4,34 +4,67 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://fonts.googleapis.com/css2?family=IM+Fell+DW+Pica+SC&family=Josefin+Sans&family=Nosifer&family=Scope+One&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css" />
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
+   <section class="main-wrapper">
+    <header class="header">
+      <section class="header-content">
+        <h1 class="title-heading">
+          Cluedo
+        </h1>
+        <blockquote class="title-quote">
+          Can you solve the murder-mystery?
+        </blockquote>
+      </section>
+    </header>
     <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
+      <div class="card-container">
+        <article class="killer-deck" id="killerCard"> 
+          <!-- removed onclick="pickKiller()" above and added eventlistener in js below pickkiller function-->
+          <p class="card-title">KILLER</p>
+          <p id="killerName"></p>
+          <p id="killerAge"></p>
+          <p id="killerOccupation"></p>
+          <p id="killerWeapon"></p>
+          <img id="killerImage" src="" height="100" />
+          <p id="killerDescription"></p>
+        </article>
+        <div class="loader" id="killerLoader"></div>
+      </div>
 
-    <h1 id="mystery"></h1>
+      <div class="card-container">
+        <article class="weapon-deck" " id="weaponCard">
+          <p class="card-title">WEAPON</p>
+          <p id="weaponName"></p>
+          <p id="weaponWeight"></p>
+          <p class="weapon-flexibility" id="weaponFlexibility"></p>
+          <img id="weaponImage" src=""/>
+        </article>
+        <div class="loader" id="weaponLoader"></div>
+      </div>
 
-    <button>REVEAL THE CRIME</button>
+      <div class="card-container">
```

Comment:
@the-j-curl I think I added the card container to be able to set position: relative to that element as it being the parent element to the loader element which has position absolute. It didn't work to set position relative om the article element since it is not a parent element to the loader.